terraform-development-factory/variables.tf

```diff
@@ -0,0 +1,120 @@
+# Provider Setup Details
+variable "harness_platform_url" {
+  type        = string
+  description = "[Optional] Enter the Harness Platform URL.  Defaults to Harness SaaS URL"
+  default     = "https://app.harness.io/gateway"
+}
+
+variable "harness_platform_account" {
+  type        = string
+  description = "[Required] Enter the Harness Platform Account Number"
+  sensitive   = true
+}
+
+variable "harness_platform_key" {
+  type        = string
+  description = "[Required] Enter the Harness Platform API Key for your account"
+  sensitive   = true
+}
+
+variable "organization_name" {
+  type        = string
+  description = "[Required] Provide an organization name.  Must be two or more characters"
+}
+
+variable "create_organization" {
+  type        = bool
+  description = "[Optional] Should this execution create a new Organization"
+  default     = false
+}
+
+variable "project_name" {
+  type        = string
+  description = "[Required] Provide an project name.  Must be two or more characters"
+}
+
+variable "repositories" {
+  type        = list(string)
+  description = "[Required] List of Repositories for which to configure and create pipelines"
+}
+
+variable "terraform_files" {
+  type        = string
+  description = "[Required] List of Repositories for which to configure and create pipelines"
```

Comment:
this desc is the same as the repo input, not sure what this wants.

Reply:
Pushed an update to the README to correct the description.

Also pushed an update to allow the harness_platform_account and harness_platform_key to be pulled from local ENV